### DISCUSSION TITLE
how to set container name in Deployment to inherit the application name ?

### DISCUSSION THREAD

**Discussion by claudio4j** [eclipse/jkube#3393] [General]
Using jkube 1.17.0 openshift maven plugin, the application is based on spring boot 3.2.2.
The container name set in the `Deployment` is set to `spring-boot`. 
Is it possible to automatically set it to the deployment name (metadata.name), but not explicitly setting `jkube.enricher.jkube-controller.name` ?

**Comment by rohanKanojia** [comment]
@claudio4j : Container name seems to be picked up from Image's alias. It's set by SpringBoot Generator to generator name `spring-boot`. You can override this behavior with `jkube.generator.spring-boot.alias` flag, it would only affect container name.

**Comment by claudio4j** [comment]
Just added this property in the pom.xml and it worked `<jkube.generator.alias>${artifactId}</jkube.generator.alias>`, thanks.
